src/args.ts

```diff
@@ -74,6 +79,11 @@ export const getYargs = async (): Promise<ReviewArgs> => {
       type: "boolean",
       default: false,
     })
+    .option("org", {
+      description: "Organization id to use for openAI",
+      type: "string",
+      default: undefined,
+    })
```

Comment:
Can you add this to the readme as well :)

Reply:
Done!